checkstyle.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0"?>
```

Comment:
Could you please move it to `./config/checkstyle` folder?

Reply:
Could you please do one-to-one copy of https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml (with all comments) and just comment out rules which we don't use? Then it's easier to spot what we don't use.

Reply:
done